auth-tokens/src/main/java/com/palantir/tokens/auth/BearerToken.java

```diff
@@ -39,6 +41,12 @@
     @JsonValue
     public abstract String getToken();
 
+    @Value.Derived
+    @Value.Auxiliary
+    byte[] getTokenAsBytes() {
```

Comment:
Any reason this is package accessible or could this be made `protected`?

Reply:
protected is actually more accessible than package?

Reply:
protected = package but also subclasses can access